liquid_node/configuration.py

```diff
@@ -72,6 +72,7 @@ class Configuration:
         'nextcloud': 'has a file share system and a contact list of users.',
         'rocketchat': 'is the old chat app; will remove shortly.',
         'matrix': 'is the new chat app',
+        'bbb': 'your visio app',
```

Comment:
visio i think is a francophone word, best call it "video conference" because "video chat" may have unprofessional connotations